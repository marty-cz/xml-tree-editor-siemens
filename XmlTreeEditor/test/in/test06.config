<config>
  <input>test01.xml</input>
  <schema>test02.xsd</schema>
  <operations>
    <operation operType="deleteNode" name="PRICE" />
    <operation type"replaceValue" oldValue="UK" newValue="GB" />
  </operations>
</config>