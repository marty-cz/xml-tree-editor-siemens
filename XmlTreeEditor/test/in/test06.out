Error: Schema file "test02.xsd" does not correspond with input xml file "test01.xml".